pages/osx/darwin-rebuild.md

```diff
@@ -0,0 +1,16 @@
+# darwin-rebuild
+
+> A command for rebuilding and switching to a Nix-based Darwin (macOS) system configuration.
+> More information: <https://github.com/LnL7/nix-darwin>.
+
+- Rebuild and switch to the specified Darwin configuration:
+
+`darwin-rebuild switch --flake {{path/to/flake}}`
+
+- Build the configuration but don't switch to it:
+
+`darwin-rebuild build --flake {{path/to/flake}}`
+
+- Show available commands and options:
```

Comment:
https://github.com/tldr-pages/tldr/blob/ab008159156b225f57d5a9ef1fd2a241d94470bd/contributing-guides/style-guide.md#help-and-version-commands
> For consistency, we prefer generic wording `Display help`
```suggestion
- Display help:
```